src/Jackett/JackettModule.cs

```diff
@@ -59,6 +59,10 @@ protected override void Load(ContainerBuilder builder)
             Mapper.CreateMap<WebClientByteResult, WebClientStringResult>().ForMember(x => x.Content, opt => opt.Ignore()).AfterMap((be, str) =>
             {
                 str.Content = Encoding.UTF8.GetString(be.Content);
+                if(str.Content.Contains("NorBits"))

```

Comment:
I think it would be better to look at the http header and see what encoding the response is being sent as.